irobot_benchmark/src/irobot_benchmark.cpp

```diff
@@ -76,7 +76,8 @@ create_ros2_system(
     static_cast<performance_test::ExecutorType>(options.executor),
     performance_test::SpinType::SPIN,
     events_output_path_opt,
-    options.csv_out);
+    options.csv_out,
+    options.timers_separate_thread);
```

Comment:
I would prefer a different name for this variable.
This name seems to indicate that we will create a new thread for timers, which is not correct (the thread is always created, the difference is what's the purpose of this thread).

Reply:
Would it make sense to keep the original name of the constructor argument `execute_timers_separate_thread` ?
Suggestions are welcome

Reply:
`execute_timers_separate_thread` is better.

a separate thread is always created;
when the flag is true, timers are monitored and executed in that thread.
when the flag is false, timers are only monitored there (and they are executed in the main executor thraed)